### PR TITLE
Added some missing logical type converters

### PR DESCRIPTION
**Pull request by arkadius**
I've added missing logical type converters for:
- `timestamp-micros`
- `date`
- `time-millis`
- `time-micros`
As it was discussed at: https://github.com/allegro/json-avro-converter/pull/72#issuecomment-1030803399
Thanks @deblockt for initial work making it possible.

### CODE REVIEW

File: converter/src/main/java/tech/allegro/schema/json2avro/converter/types/IntDateConverter.java [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+import java.util.Deque;
+
+import static org.apache.avro.Schema.Type.INT;
+import static tech.allegro.schema.json2avro.converter.PathsPrinter.print;
+import static tech.allegro.schema.json2avro.converter.types.AvroTypeConverter.isLogicalType;
+
+public class IntDateConverter implements AvroTypeConverter {
```
Comment (deblockt) [inline converter/src/main/java/tech/allegro/schema/json2avro/converter/types/IntDateConverter.java:16]: Maybe that we can provide an abstract class or an util to merge all date parsing code ?

Comment (arkadius) [reply converter/src/main/java/tech/allegro/schema/json2avro/converter/types/IntDateConverter.java]: Good point :+1:  I've extracted abstract classes

**Comment by arkadius** [comment]
Hi guys @szczygiel-m , @piotrrzysko . Can u take a look at this in a free time? Thanks in advance.

File: converter/src/main/java/tech/allegro/schema/json2avro/converter/CompositeJsonToAvroReader.java [hunk 2]
```diff
@@ -3,16 +3,7 @@
...
-import tech.allegro.schema.json2avro.converter.types.BytesDecimalConverter;
-import tech.allegro.schema.json2avro.converter.types.EnumConverter;
-import tech.allegro.schema.json2avro.converter.types.LongTimestampMillisConverter;
-import tech.allegro.schema.json2avro.converter.types.MapConverter;
-import tech.allegro.schema.json2avro.converter.types.NullConverter;
-import tech.allegro.schema.json2avro.converter.types.PrimitiveConverter;
-import tech.allegro.schema.json2avro.converter.types.RecordConverter;
-import tech.allegro.schema.json2avro.converter.types.AvroTypeConverter;
-import tech.allegro.schema.json2avro.converter.types.UnionConverter;
+import tech.allegro.schema.json2avro.converter.types.*;
```
[the remark this answers is not in the document]
Comment (arkadius) [reply converter/src/main/java/tech/allegro/schema/json2avro/converter/CompositeJsonToAvroReader.java]: sure, done

[the remark this answers is not in the document]
Comment (arkadius) [reply converter/src/main/java/tech/allegro/schema/json2avro/converter/CompositeJsonToAvroReader.java]: No problem, thanks for the review

**Pull request merged**